src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseAnalyzers.cs

```diff
@@ -50,7 +50,7 @@ public void It_resolves_configbindinggenerator_correctly(string testAssetName, b
                 });
 
             VerifyConfigBindingGeneratorIsUsed(asset, isEnabled);
-            VerifyInterceptorsFeatureEnabled(asset, isEnabled);
+            VerifyInterceptorsFeatureEnabled(asset, isEnabled, "Microsoft.Extensions.Configuration.Binder.SourceGeneration");
```

Comment:
nit: these `Verify*` method names read like they (always) assert that the generator(s) are used/enabled. Consider using names that indicate that the expected behavior depends on whether the generator is enabled (e.g VerifyInterceptorsFeatureSettingIsCorrect).